node/src/accountant/payment_adjuster/non_unit_tests/mod.rs

```diff
@@ -576,6 +598,35 @@ fn merge_information_about_particular_account(
         .collect()
 }
 
+fn prepare_interpretable_adjustment_results(
+    merged: Vec<(AccountInfo, Option<PayableAccount>)>,
+) -> (Vec<InterpretableAccountAdjustmentResult>, usize) {
+    merged.into_iter().fold(
+        (vec![], 0),
+        |(mut interpretable_accounts, adjusted_accounts_so_far),
+         (info, maybe_eliminated_account)| {
+            let (interpretable_account, was_this_account_eliminated) =
+                InterpretableAccountAdjustmentResult::new(info, maybe_eliminated_account);
+            interpretable_accounts.push(interpretable_account);
+            (
+                interpretable_accounts,
+                adjusted_accounts_so_far + if was_this_account_eliminated { 1 } else { 0 },
+            )
+        },
+    )
+}
+
+fn look_after_adjustment_statistics(
+    test_overall_output_collector: &mut TestOverallOutputCollector,
+    adjusted_accounts_within_this_scenario: usize,
+) {
+    if adjusted_accounts_within_this_scenario > 1 {
+        test_overall_output_collector.scenarios_with_more_than_one_adjustment += 1
+    }
+    test_overall_output_collector.adjusted_account_count_total +=
+        adjusted_accounts_within_this_scenario;
+}
```

Comment:
I don't know why this specific statistic is required, if you could answer `Why it exist?`, that'll be great.